package.json

```diff
@@ -3,6 +3,9 @@
   "version": "3.5.0",
   "private": true,
   "license": "Apache-2.0",
+  "resolutions": {
+    "semver": "7.5.3"
```

Comment:
wondering why is not 
```
"resolutions": {
    "cls-hooked" : {
        "semver": "7.5.3"
    }
}
```

Reply:
I tried that initially but it seems that `npm-force-resolutions` doesn't support specifying the direct dependency -> transitive dependency. When I ran `lerna bootstrap --hoist` to install,  the `package-lock.json` was not updated correctly in this case.

Reply:
Good to know